setup.cfg

```
@@ -20,6 +20,9 @@ install_requires =
 [options.extras_require]
 s3 =
   boto3==1.24.93
+  openpyxl==3.0.9
+  shapely==2.0.2
+  geopandas==0.13.2
```

It may be best to add a `gis` environment to include `shapely` and `geopandas`
I'd suggest adding `openpyxl` to another environment `io_extras`, or add to install_requires

```suggestion
gis =
  shapely==2.0.2
  geopandas==0.13.2
io_extras = 
  openpyxl==3.0.9
```

---

Could we just add pandas[excel]==1.5.1 ? And actually thinking about it, geopandas has shapely as a dependency, so can just remove it right?

---

Yep!